dev/src/Composer.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * Copyright 2017 Google Inc.
```

Comment:
```suggestion
 * Copyright 2023 Google Inc.
```

Reply:
This file was actually moved from a different file, so I'd prefer to keep this one the same.